apps/judicial-system/api/infra/judicial-system-api.ts

```diff
@@ -7,15 +7,20 @@ export const serviceSetup = (services: {
     .namespace('judicial-system')
     .serviceAccount('judicial-system-api')
     .env({
-      SAML_ENTRY_POINT: {
-        dev: 'https://innskraning.island.is/?id=judicial-system.dev',
-        staging: 'https://innskraning.island.is/?id=judicial-system.staging',
-        prod: 'https://innskraning.island.is/?id=rettarvorslugatt.prod',
+      AUTH_IDS_SCOPE: 'openid profile',
+      AUTH_IDS_CLIENT_ID: '@rettarvorslugatt.island.is/web',
+      AUTH_IDS_DOMAIN: {
+        dev: 'https://identity-server.dev01.devland.is',
+        staging: 'https://identity-server.staging01.devland.is',
+        prod: 'https://innskra.island.is',
       },
-      AUTH_AUDIENCE: {
-        dev: ref((h) => `judicial-system.${h.env.domain}`),
-        staging: ref((h) => `judicial-system.${h.env.domain}`),
```

Comment:
I'm not too fond of hard-coded URLs. This change will probably break feature-deployments.

Reply:
Could you elaborate, do you mean rather doing something like this? :
 dev: ref(
          (h) =>
            `https://judicial-system.${h.env.domain}/api/auth/callback/identity-server`,
        ),
        
        Or do you mean they should be kept somewhere else like the AWS parameter store?

Reply:
Yes, that should work. Can we try it out and make a feature deployment to be sure?